CHANGES.rst

```diff
@@ -5,6 +5,7 @@ ChangeLog
 | Version    | Description                                                         | Date       |
 +============+=====================================================================+============+
 | **2.5.0**  | * Add HD44780 LCD display driver                                    | TBA        |
```

Comment:
I thought we'd released this?
Did i not update the date?!?

Reply:
hmm.. you tell me :) What should i change here?

Reply:
ps. this PR needs to be force merged because the json aren't updated yet.

Reply:
Fixed it